src/isReply.js

```diff
@@ -0,0 +1,15 @@
+const isReply = (tweet) => {
+    var RT = /^RT/i;
+    
+    if (RT.test(tweet.text) ||
+        tweet.is_quote_status ||
+        tweet.retweeted_status ||
+        tweet.in_reply_to_status_id ||
+        tweet.in_reply_to_status_id_str ||
+        tweet.in_reply_to_user_id ||
+        tweet.in_reply_to_user_id_str ||
+        tweet.in_reply_to_screen_name)
+        return true;
```

Comment:
Formatting isn't in line with rest of project, I can clean this up if you like 

I have a PR I'm waiting on some changes for but the Prettier formatting rule I'd prefer for here is:

```json
"prettier es5 --no-semi --single-quote --write src/**/*.js"
```

In your change it's only semicolons need removing